pandas/tests/indexes/test_multi.py

```diff
@@ -1457,7 +1486,16 @@ def test_equals_multi(self):
 
         index = MultiIndex(levels=[major_axis, minor_axis],
                            labels=[major_labels, minor_labels])
-        self.assertFalse(self.index.equals(index))
+        assert not self.index.equals(index)
+
+    def test_equals_missing_values(self):
+        # make sure take is not using -1
+        i = pd.MultiIndex.from_tuples([(0, pd.NaT),
+                                       (0, pd.Timestamp('20130101'))])
+        result = i[0:1].equals(i[0])
```

Comment:
I assume this is the test related to my other comment. So what I also said there, is that this equivalence is not true for single-level indices:

```
In [15]: i = pd.Index([1,2,3])

In [16]: i[0:1]
Out[16]: Int64Index([1], dtype='int64')

In [17]: i[0:1].equals(i[0])
Out[17]: False
```

So why making this different for MultiIndex ?